### PR TITLE
[CS2113T-T09-1]-ngjiewu-Refactor UI 

### PR DESCRIPTION
**Pull request by ngjiewu**
- Recurring tasks
- Moved most of the printing from TaskList.java to UI.java

### CODE REVIEW

File: src/main/java/seedu/hustler/task/TaskList.java [hunk 1]
```diff
@@ -85,14 +76,9 @@ public void add(Task task) {
             AddTask.updateAchievementLevel();
             AddTask.updatePoints();
             AchievementList.updateAddTask(addAchievementLevel);
-            String output = "\t  " + list.get(list.size() - 1).toString();
-            System.out.println("\t_____________________________________");
-            System.out.println("\tGot it. I've added this task:");
-            System.out.println(output);
-            System.out.println("\tNow you have " + list.size() + " tasks in the list.");
-            System.out.println("\t_____________________________________\n\n");
+            ui.show_task_added(list);
```
Comment (yzia2000) [inline src/main/java/seedu/hustler/task/TaskList.java:85]: Great job making it more abstract

File: src/main/java/seedu/hustler/task/TaskList.java [hunk 2]
```diff
@@ -85,14 +76,9 @@ public void add(Task task) {
...
         } else {
-            System.out.println("Task clashes with another existing task in the list!");
+            ui.show_task_clash();
```
Comment (yzia2000) [inline src/main/java/seedu/hustler/task/TaskList.java:87]: good idea

File: src/main/java/seedu/hustler/task/TaskList.java [hunk 3]
```diff
@@ -137,30 +123,8 @@ public void add(String taskType, String taskDescriptionFull) {
...
-                        numOfMin = number * 60 * 24 * 7;
-                        break;
-                    case "months":
-                        numOfMin = number * 60 * 24 * 7 * 28;
-                        break;
-                    default:
-                        numOfMin = 0;
-                        break;
-                    }
+                    int numOfMin = convertToMinute(frequency);
```
Comment (yzia2000) [inline src/main/java/seedu/hustler/task/TaskList.java:132]: definitely much required

**Review by yzia2000 — APPROVED** [review]
Well done. Much needed changes

**Review by nystera — APPROVED** [review]

**Pull request merged**